src/components/prelude/auth/mod.rs

```diff
@@ -19,6 +20,11 @@ pub struct Props {
     account: Account,
 }
 
+fn remove_whitespace(s: &mut String) -> String {
```

Comment:
If youre going to use this you should either mutate the borrowed string or return. Not both. eg

```rust
fn remove_writespace(s: &mut String) {
    s.retain(|c| !c.is_whitespace());
}
```

or 

```rust
fn remove_writespace(s: String) -> String {
    let mut s = s.clone();
    s.retain(|c| !c.is_whitespace());
    s
}
```

Might be better to use `&str` for the latter